### DISCUSSION TITLE
posts/run-gpu-jobs-on-slurm/

### DISCUSSION THREAD

**Discussion by giscus** [legendPerceptor/legendPerceptor.github.io#2] [General]
# posts/run-gpu-jobs-on-slurm/

First of all, when you are on a login node, you usually have no access to a GPU. If you want to test PyTorch with a GPU environment, it is better to enter a GPU session first.

https://www.yuanjianliu.net/posts/run-gpu-jobs-on-slurm/

**Comment by legendPerceptor** [comment]
For the CIFAR10 dataset, it is easier to work with by setting the download to be true on your local machine and then copy the data to your desired supercomputer.
